templates/taxi/driver-confirm-delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
```

Comment:
indentation in 2 indents is broken in html files, please fix it

Reply:
i probably have this bug with html again
here is Pycharm view:
![image](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/assets/141722835/77377733-064d-477c-9104-a39e8d83a908)

here is HTML settings:
![image](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/assets/141722835/8b78cef2-5110-4932-9120-9f6d521e7225)

hope last commit will fix it